src/instance/DataModelInstance.ts

```diff
@@ -37,6 +37,18 @@ export default class DataModelInstance {
     return [...this.root.values()][0].toJSON(camelCase);
   }
 
+  public toXML(rootEl?: Element) {
+    if (rootEl) {
+      [...this.root.values()][0].toXML(rootEl);
```

Comment:
Does this pay a performance penalty to expand the iterable into an array just to pick the first item? Could there be a lot?